src/routes/v2/address.ts

```diff
@@ -18,6 +18,13 @@ util.inspect.defaultOptions = { depth: 1 }
 const BITBOXCli = require("bitbox-sdk/lib/bitbox-sdk").default
 const BITBOX = new BITBOXCli()
 
+// Max number of items per request for freemium access.
+const FREEMIUM_INPUT_SIZE = 20
+
+// Use the default (and max) page size of 1000
+// https://github.com/bitpay/insight-api#notes-on-upgrading-from-v03
+const PAGE_SIZE = 1000
```

Comment:
Needed to move `pageSize` value to a higher scope so that it could be used outside of `detailsFromInsight()`